sqlstore/src/main/resources/db/migration/V0261__auto_site.sql

```diff
@@ -3,19 +3,43 @@
 -- StartNoTest
 INSERT INTO SampleClass(alias, sampleCategory, suffix, createdBy, creationDate, updatedBy, lastUpdated, dnaseTreatable)
   VALUES ('Tissue', 'Tissue', NULL, (SELECT userId FROM User WHERE loginName = 'admin'), CURRENT_TIMESTAMP, (SELECT userId FROM User WHERE loginName = 'admin'), CURRENT_TIMESTAMP, FALSE);
+
 SET @tissueId = LAST_INSERT_ID();
 
 ALTER TABLE TissueType ADD COLUMN sampleTypeName varchar(255);
 
 -- This query will fail if there's more than one sample type for each tissue class. This is by design. Those records need to be fixed.
-UPDATE TissueType SET sampleTypeName = (SELECT DISTINCT alias FROM SampleClass JOIN DetailedSample USING (sampleClassId) JOIN SampleTissue USING (sampleId) WHERE sampleCategory = 'Tissue' AND SampleTissue.tissueTypeId = TissueType.tissueTypeId);
+UPDATE TissueType SET sampleTypeName = (
+  SELECT DISTINCT alias
+  FROM SampleClass
+  JOIN DetailedSample USING (sampleClassId)
```

Comment:
Was @apmasell, not me. I'd avoid `USING` because I don't think it's included in H2 grammar. It is okay for "NoTest" stuff though